src/glyph/serialize.rs

```diff
@@ -79,13 +79,15 @@ impl Glyph {
         // glyph.lib in-memory. If there are object libs to serialize, clone the
         // existing lib and insert them there for serialization, otherwise avoid
         // cloning and write out the original.
+        let mut lib = self.lib.clone();
         let object_libs = self.dump_object_libs();
         if !object_libs.is_empty() {
-            let mut new_lib = self.lib.clone();
-            new_lib.insert(PUBLIC_OBJECT_LIBS_KEY.into(), plist::Value::Dictionary(object_libs));
-            write_lib_section(&new_lib, &mut writer)?;
-        } else if !self.lib.is_empty() {
-            write_lib_section(&self.lib, &mut writer)?;
+            lib.insert(PUBLIC_OBJECT_LIBS_KEY.into(), object_libs.into());
```

Comment:
The changes as of https://github.com/linebender/norad/pull/147/commits/1ee9066f31ff21a5e44d48b903ad092dbbdb41b8 seem to be writing empty `public.objectLibs` dictionary fields in the lib.plist files for some reason

https://github.com/chrissimpkins/ufo-format-diff/commit/6633c6bc2a9f8a0f8527c4383510068d3ed6a10c